src/v2/models/AppState.js

```diff
@@ -106,6 +106,28 @@ export default Model.extend({
     return currentViewState;
   },
 
+  /**
+   * Checks to see if we're in an enrollment or challenge flow.
```

Comment:
@gowthamidommety-okta - Thanks for the ping yesterday about this.

There are two cases we want to account for:

1. The authenticator supports only one enrollment (email, duo, etc)
2. The authenticator supports multiple enrollments

As a result, when we're determining whether or not we're in a challenge flow - we can look into both of these objects for verification.